### PR TITLE
Add Gen UTCTime to avoid aeson asymmetry

### PR DESCRIPTION
**Pull request by charleso**

### CODE REVIEW

File: orphanarium-aeson/src/Orphanarium/Aeson/Time.hs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import           Data.Time
+
+import           Test.QuickCheck
+
+
+genTime :: Gen UTCTime
+genTime = UTCTime
+  <$> fmap ModifiedJulianDay arbitrary
+  <*> fmap secondsToDiffTime (choose (0, 60 * 60 * 24))
```
Comment (charleso) [inline orphanarium-aeson/src/Orphanarium/Aeson/Time.hs]: Anything higher than that and it fails

Comment (markhibberd) [inline orphanarium-aeson/src/Orphanarium/Aeson/Time.hs]: What about millis? Maybe we don't care - but they should work.

**Comment by markhibberd** [comment]
:+1:

File: orphanarium-aeson/src/Orphanarium/Aeson/Time.hs (hunk 1 above)
Comment (charleso) [inline orphanarium-aeson/src/Orphanarium/Aeson/Time.hs]: > What about millis?

`fmap secondsToDiffTime (choose (0, 60 * 60 * 24 * 1000))`

    Left "could not parse ISO-8601 date"

**Pull request merged**